### PR TITLE
Update CHANGELOG for 1.5.0-rc.1

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Update CHANGELOG for 1.5.0-rc.1

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Prometheus.HttpListener/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.0-rc.1
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Exporter.Prometheus.HttpListener/CHANGELOG.md:5]: remove?

File: src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.0-rc.1
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md:5]: same as https://github.com/open-telemetry/opentelemetry-dotnet/pull/4519/files#r1206076638

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.0-rc.1
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/CHANGELOG.md:5]: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4519/files#r1206076747

**Review by vishweshbankwar — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Console/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.0-rc.1
+
+Released 2023-May-25
```
Comment (alanwest) [inline src/OpenTelemetry.Exporter.Console/CHANGELOG.md:7]: Oops, I forgot to add a changelog to for #4507

```suggestion
Released 2023-May-25

* Add display of bucket boundaries when using exponential histograms.
   ([#4507](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4507))
```

**Review by alanwest — APPROVED** [review]

**Pull request merged**